pkgs/development/python-modules/yaspin/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, callPackage
+, fetchPypi
+, buildPythonPackage
+, termcolor
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "yaspin";
+  version = "2.1.0";
```

Minimal support Python release is [3.6](https://github.com/pavdmyt/yaspin/blob/master/pyproject.toml#L50).

```suggestion
  version = "2.1.0";

    disabled = pythonOlder "3.6";
```